testsuite/forge.py

```diff
@@ -2023,12 +1866,17 @@ def set_config(
     for k, v in values:
         config.set(k, v, validate=not force)
 
-    config.flush()
+    if preview:
```

Comment:
neato, can you add the ability to preview then confirm if the value looks good?

That might be a nice default